.gitignore

```diff
@@ -58,10 +58,12 @@ target/
 .idea/*
 *~
 scbU
+
 npm-debug.log
 .DS_Store
 html_app/index.html
 
-
 .vagrant/
 static/
+
+
```

Comment:
It would be nice to clean up these blank lines, but I don't think it's worth the rebase right now.